pandas/core/generic.py

```diff
@@ -8650,6 +8656,50 @@ def tz_localize(self, tz, axis=0, level=None, copy=True,
         ------
         TypeError
             If the TimeSeries is tz-aware and tz is not None.
+
+        Examples
+        --------
+
+        Localize local times:
```

Comment:
For this set of examples, I think these methods should be showing examples for `Series.tz_localize` (the timeseries data is passed to `index=`) and not `Series.dt.tz_localize`(the timeseries data is pass to `data=` since that's where this docstring is linked